package.json

```diff
@@ -186,8 +186,8 @@
     "last 1 years"
   ],
   "engines": {
-    "node": "24.12.0",
-    "pnpm": "10.25.0"
+    "node": "^24.12.0",
+    "pnpm": "^10.25.0"
```

Comment:
To handle

```
 ERR_PNPM_UNSUPPORTED_ENGINE  Unsupported environment (bad pnpm and/or Node.js version)

Your pnpm version is incompatible with "tmp/crates.io".

Expected version: 10.23.0
Got: 10.25.0

This is happening because the package's manifest has an engines.pnpm field specified.
To fix this issue, install the required pnpm version globally.

To install the latest version of pnpm, run "pnpm i -g pnpm".
To check your pnpm version, run "pnpm -v".

Your Node version is incompatible with "tmp/crates.io".

Expected version: 24.11.1
Got: v24.12.0

This is happening because the package's manifest has an engines.node field specified.
To fix this issue, install the required Node version.
```

Reply:
the "engines" are explicitly pinned because we use those fields to determine which versions to use during the deploy process. we are aware that this a little annoying at the moment, but this is a change that needs a bit more design before we can unpin them.

Reply:
https://github.com/rust-lang/crates.io/pull/12538 should hopefully help fix this issue